docs/01-prerequisites.md

```diff
@@ -113,6 +113,11 @@ find it and, if you continue with the installation without selecting a key,
 you will not be able to select to use this SSH key with your instances, and
 you will not be able to SSH into them at all.
 
+Regardless of the SSH key (or lack thereof) chosen at install-time,
+you can [adjust your deployed SSH
+keys][machine-config-daemon-ssh-keys] via the machine-config operator
+at any point after the cluster comes up.
+
```

Comment:
If the operator can be used to add SSH keys to hosts then I'd prefer these two paragraphs be modified to say that "without using the operator" you wouldn't be able to get to the hosts.

Reply:
I'd have expected us to want to emphasize "you can manage this on day 2, like (almost) everything else".

Reply:
Yes. Both?

Reply:
How about:

The installer will interactively prompt for an SSH keypair if it finds one in ~/.ssh/ that ends with .pub. If no .pub file is found, the installer will not prompt for an SSH keypair and will install the cluster without SSH access.

If you install a cluster without initially selecting an SSH keypair, you can still add SSH keypairs to the hosts using the machine-config operator.

?

Reply:
Sounds good, thanks.  I'll reroll once I clock in.

Reply:
Ok, I've pushed 7e5f974 -> 9c4f337, rebasing onto master and adjusting the wording to be closer to your proposal above.  Differences include using "public key" instead of "keypair" (because the installer doesn't care about private SSH keys) and preserving the arrow-navigation hint from the previous content.

I've also added a note to the commit message about the "only one `~/.ssh/*.pub`" special case, but as I say there, I don't think that's important enough to call out in the docs themselves.

Reply:
And I've pushed 9c4f337 -> bdee07f, consolidating the previous note content with the main subsection content.  I think that makes sense, but if you want to shuffle things differently, feel free to close this PR and make the changes yourself ;).